lib/api_movie.dart

```diff
@@ -0,0 +1,101 @@
+import '../constants/strings.dart' as strings;
+
+import 'api_genres.dart';
+import 'genre.dart';
+import 'movie.dart';
+
+class ApiMovie {
+  final bool adult;
+  final double popularity;
+  final int id;
+  final int voteCount;
+  final String title;
+  final String originalTitle;
+  final bool video;
+  final List<dynamic> genres;
```

Comment:
This list is not dynamic, is a `List<int>`

Reply:
This change was already applied in previous commits